perk-store-services/src/main/java/org/exoplatform/perkstore/service/PerkStoreService.java

```diff
@@ -333,6 +333,17 @@ public Product saveProduct(Product product, String username) throws Exception {
     return orders;
   }
 
+  public List<ProductOrder> getUserOrders(String username) {
+
+    if (StringUtils.isBlank(username)) {
+      throw new IllegalArgumentException("username is mandatory");
+    }
+    List<ProductOrder> orders =perkStoreStorage.getUserOrders(username);;
```

Comment:
Make sure to format well the code.
```suggestion
    List<ProductOrder> orders = perkStoreStorage.getUserOrders(username);
```